lib/Optimizer/GraphOptimizer.cpp

```diff
@@ -951,6 +951,85 @@ static void optimizeQuantization(Function *F) {
   optimizeQuantizedMaxSplat(F);
 }
 
+/// Sink Rescale nodes down when possible.
+static bool sinkRescaleQuantizedNode(Function *F) {
+  bool changed = false;
+  for (auto const &node : F->getNodes()) {
+    // Sink Rescale below Reshape node.
```

Comment:
Can you add a comment with the formula that explains what's the transformation:
// reshape(rescale(X))  -> rescale(reshape(X))

Reply:
fixed.